### PR TITLE
Removed low-value browser tests

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
I removed a lot of browser tests that were a bit fragile. Here are the cases were I removed tests.
- If the test was testing if a string existed
- If the test didn't test any logic in the template
- If the test could be broken by the CMS alone

I updated a few tests to include missing stories from the activity log and adjust for adding `- Consumer Financial Protection Bureau` to all our titles.

## Testing

- `gulp test:acceptance` or `gulp test:acceptance --sauce=false`

## Review

- @anselmbradford 
- @jimmynotjim 
- @sebworks 

## Todos

- There's a lot of things that could be tested on wagtail pages, but since we don't have a good way to match what we have locally to beta, this is hard to do. Additionally we run into the same problems of depending on CMS generated content that can break tests.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: test/browser_tests/page_objects/page_careers.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 
 var _getQAelement = require( '../util/qa-element' ).get;
-var careersSocialSection =
-require( '../shared_objects/careers-social-section' );
-
 
 function Careers() {
 
-  Object.assign( this, careersSocialSection );
+  Object.assign( this );
```
Comment (sebworks) [inline test/browser_tests/page_objects/page_careers.js]: We can remove this line.

**Comment by sebworks** [comment]
:+1: , I would remove `qa-hooks` from the templates as well if they are being deleted from the tests.

**Comment by jimmynotjim** [comment]
:+1: When everything is moved to wagtail, we might just have to check items exist, or the text length is greater than zero since we won't be able to guarantee the content.

**Comment by KimberlyMunoz** [comment]
Would it make sense to check high-traffic pages that don't change often for functional things, like expandables working and multi-selects generating properly.

File: test/browser_tests/page_objects/page_careers.js (hunk 1 above)
Comment (KimberlyMunoz) [inline test/browser_tests/page_objects/page_careers.js]: Thanks! Fixed.

**Comment by KimberlyMunoz** [comment]
@sebworks Good looking out on the extra unused qa-hooks. Removed in 033a1ec

**Comment by sebworks** [comment]
:+1: , @KimberlyMunoz.

**Pull request merged**